sci-mathematics/gimps/gimps-30.5.2.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit pax-utils readme.gentoo-r1 systemd tmpfiles
+
+DESCRIPTION="The Great Internet Mersenne Prime Search"
+HOMEPAGE="https://www.mersenne.org/"
+
+# The author has finally caved in to the pressure of putting the build
+# number in the package name. Some dark magic necessary to get to the
+# package name
+
+MY_PV=$(ver_rs 2 'b' )
+
+SRC_URI="
+	amd64? ( https://www.mersenne.org/ftp_root/gimps/p95v${MY_PV/./}.linux64.tar.gz )
+	x86? ( https://www.mersenne.org/ftp_root/gimps/p95v${MY_PV/./}.linux32.tar.gz )
```

Comment:
For the next version bump, can we rename this to be more identifiable among the distfiles? Just add `-> ${P}.linux64.tar.gz` etc.